### ISSUE TITLE
All but one favourites gone after app becoming unresponsive

### ISSUE THREAD

**Issue by dafadllyn** [osmandapp/OsmAnd#15487]
While i wanted to save a new favourite, OsmAnd became unresponsive. After closing and reopening the app, all favourites but the new one were gone. (Thanks to the backups the app creates i could recover all favourites.)

I'm using OsmAnd+ version 4.2.7 (installed from the Google Play Store) on a Nokia 6.1 running Android 10.

Here are exception.log and logcat.log, although i couldn't find anything useful in them (the problem occurred today 2022-10-16 at around 14:15 UTC+02:00):

[exception.log](https://github.com/osmandapp/OsmAnd/files/9794829/exception.log)
[logcat.log](https://github.com/osmandapp/OsmAnd/files/9794830/logcat.log)

**Comment by tradavyvy** [comment]
Link to #15333 ?
Losing favourites is always a disaster, they represent lots of work and valuable information.
Of course backupshelp, but still .....

**Comment by vshcherb** [comment]
Favorites backups might be accessible still in backups.

**Comment by dafadllyn** [comment]
> Favorites backups might be accessible still in backups.

Yes, luckily i was able to restore the favourites this way. For the less tech-savvy, however, it would be good if the favourites were not lost at all. 😁
